docs/developer/extending/apps.mdx

```diff
@@ -1,8 +1,338 @@
+| `ORDER_CREATED`             | A new order is placed.                                                                                                           |
+| `ORDER_FULLY_PAID`          | Payment is made and an order is fully paid.                                                                                      |
+| `ORDER_UPDATED`             | An order is updated; triggered for all changes related to an order; covers all other order webhooks, except for `ORDER_CREATED`. |
+| `ORDER_CANCELLED`           | An order is cancelled.                                                                                                           |
+| `ORDER_FULFILLED`           | An order is fulfilled.                                                                                                           |
+| `FULFILLMENT_CREATED`       | A new fulfillment is created.                                                                                                    |
+| `PRODUCT_CREATED`           | A new product is created.                                                                                                        |
+
+Webhooks' payloads are sent in `POST` requests.
+
+### Type of Apps
+- `local`: create a custom App instance, assign explicitly provided permissions. 
+The app is fully manageable from dashboard. You can change assigned permissions, add webhooks, or authentication token. 
+`local` app can be useful for creating integration with some services which will parse the webhook payload.  
+:::note
+If you're building a single service dedicated only to your shop this solution will handle all your needs.
+:::
+
+- `third-party`: install App in Saleor and proceed with all required steps to prepare the communication channel for App.  
+Installation is fully automated.  Saleor uses a defined App manifest to gather all required information.  
+If there are no errors during the installation, App gets the possibility to attach to Saleor over the iframe, 
+communicate with Saleor over API and use webhooks to subscribe for the given events.
+:::note
+If you're building an App that will be published in the marketplace, then this solution is dedicated to you. 
+:::
+
+## Creating and installing Apps from the command line
+
+Saleor provides commands that can be used to create/install an App from the command line.
+### `create_app`
+
+Creates a `local` App with explicitly provided permissions.
+
+Arguments:
+
+- `<name>`: the name of the App, required.
+- `--permission <PERM>`: assign permission `PERM` to the App. This argument can be repeated to specify multiple permissions.
+- `--activate`: immediately activate the new App.
+- `--target-url <URL>`: URL to send the API auth token to once installation is complete.
+
+> **Example:** The following command will create an App named "Order App" that can access users and orders:
+>
+> ```shell
+> python manage.py create_app "Order App" \
```

Comment:
What is more common, using the target URL or not? I think it would be useful if this example showed how to use `--target-url`. Then you would put the "If the target URL is provided" part right after this example to explain it. The "If the target URL is not specified" part would go later.